pupa/scrape/schemas/person.py

```diff
@@ -16,6 +16,7 @@
         "national_identity": { "type": "string", "blank": True },
         "contact_details": contact_details,
         "links": links,
+        "source_identified": { "type": ["null", "boolean"] },
```

Comment:
Something strikes me as funny about having this on people and orgs.

Perhaps we can set it on a per-jurisdiction basis. Thoughts, @boblannon ?